### ISSUE TITLE
The example fails to install

### ISSUE THREAD

**Issue by johnjelinek** [unindented/electron-installer-windows#16]
```
2016-03-21 12:48:54> Program: Starting Squirrel Updater: --install .
2016-03-21 12:48:54> Program: Starting install, writing to C:\Users\IEUser\AppData\Local\SquirrelTemp
2016-03-21 12:48:54> Program: About to install to: C:\Users\IEUser\AppData\Local\poopie
2016-03-21 12:48:54> CheckForUpdateImpl: Failed to load local releases, starting from scratch: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\IEUser\AppData\Local\poopie\packages\RELEASES'.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   at Squirrel.Utility.LoadLocalReleases(String localReleaseFile)
   at Squirrel.UpdateManager.CheckForUpdateImpl.<CheckForUpdate>d__2.MoveNext()
2016-03-21 12:48:54> CheckForUpdateImpl: Reading RELEASES file from C:\Users\IEUser\AppData\Local\SquirrelTemp
2016-03-21 12:48:54> CheckForUpdateImpl: First run or local directory is corrupt, starting from scratch
2016-03-21 12:48:54> ApplyReleasesImpl: Writing files to app directory: C:\Users\IEUser\AppData\Local\poopie\app-0.0.1
2016-03-21 12:48:57> ApplyReleasesImpl: Moving file C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\lib\net45\content_resources_200_percent.pak to C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\content_resources_200_percent.pak
2016-03-21 12:48:57> ApplyReleasesImpl: Moving file C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\lib\net45\content_shell.pak to C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\content_shell.pak
2016-03-21 12:48:57> ApplyReleasesImpl: Moving file C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\lib\net45\d3dcompiler_47.dll to C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\d3dcompiler_47.dll
2016-03-21 12:48:57> ApplyReleasesImpl: Moving file C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\lib\net45\ffmpeg.dll to C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\ffmpeg.dll
2016-03-21 12:48:57> ApplyReleasesImpl: Moving file C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\lib\net45\icudtl.dat to C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\icudtl.dat
2016-03-21 12:48:57> ApplyReleasesImpl: Moving file C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\lib\net45\libEGL.dll to C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\libEGL.dll
2016-03-21 12:48:57> ApplyReleasesImpl: Moving file C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\lib\net45\libGLESv2.dll to C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\libGLESv2.dll
2016-03-21 12:48:57> ApplyReleasesImpl: Moving file C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\lib\net45\LICENSE to C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\LICENSE
2016-03-21 12:48:57> ApplyReleasesImpl: Moving file C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\lib\net45\msvcp120.dll to C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\msvcp120.dll
2016-03-21 12:48:57> ApplyReleasesImpl: Moving file C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\lib\net45\msvcr120.dll to C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\msvcr120.dll
2016-03-21 12:48:57> ApplyReleasesImpl: Moving file C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\lib\net45\natives_blob.bin to C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\natives_blob.bin
2016-03-21 12:48:57> ApplyReleasesImpl: Moving file C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\lib\net45\node.dll to C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\node.dll
2016-03-21 12:48:57> ApplyReleasesImpl: Moving file C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\lib\net45\poopie.exe to C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\poopie.exe
2016-03-21 12:48:57> ApplyReleasesImpl: Moving file C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\lib\net45\snapshot_blob.bin to C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\snapshot_blob.bin
2016-03-21 12:48:57> ApplyReleasesImpl: Moving file C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\lib\net45\squirrel.exe to C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\squirrel.exe
2016-03-21 12:48:57> ApplyReleasesImpl: Moving file C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\lib\net45\xinput1_3.dll to C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\xinput1_3.dll
2016-03-21 12:48:57> ApplyReleasesImpl: Moving file C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\lib\net45\ui_resources_200_percent.pak to C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\ui_resources_200_percent.pak
2016-03-21 12:48:57> ApplyReleasesImpl: Moving file C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\lib\net45\vccorlib120.dll to C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\vccorlib120.dll
2016-03-21 12:48:58> ApplyReleasesImpl: Squirrel Enabled Apps: [C:\Users\IEUser\AppData\Local\poopie\app-0.0.1\poopie.exe]
2016-03-21 12:48:58> ApplyReleasesImpl: Starting fixPinnedExecutables
2016-03-21 12:48:58> ApplyReleasesImpl: Examining Pin: Cmder.lnk
2016-03-21 12:48:58> ApplyReleasesImpl: Examining Pin: File Explorer.lnk
2016-03-21 12:48:58> ApplyReleasesImpl: Examining Pin: Visual Studio 2015.lnk
2016-03-21 12:48:58> ApplyReleasesImpl: Examining Pin: Windows PowerShell.lnk
2016-03-21 12:48:58> ApplyReleasesImpl: Fixing up tray icons
2016-03-21 12:48:58> ApplyReleasesImpl: Couldn't rewrite shim RegKey, most likely no apps are shimmed: System.NullReferenceException: Object reference not set to an instance of an object.
   at Squirrel.UpdateManager.ApplyReleasesImpl.<unshimOurselves>b__13_0(RegistryView view)
2016-03-21 12:48:58> ApplyReleasesImpl: Couldn't rewrite shim RegKey, most likely no apps are shimmed: System.NullReferenceException: Object reference not set to an instance of an object.
   at Squirrel.UpdateManager.ApplyReleasesImpl.<unshimOurselves>b__13_0(RegistryView view)
2016-03-21 12:48:58> ApplyReleasesImpl: cleanDeadVersions: for version 0.0.1
2016-03-21 12:48:58> ApplyReleasesImpl: cleanDeadVersions: exclude folder app-0.0.1
2016-03-21 12:48:59> IEnableLogger: Failed to create uninstaller registry entry: System.Xml.XmlException: 'Default' tag requires a nonempty 'Extension' attribute. Line 10, position 4.
   at System.IO.Packaging.ZipPackage.ContentTypeHelper.ValidateXmlAttribute(String attributeName, String attributeValue, String tagName, XmlTextReader reader)
   at System.IO.Packaging.ZipPackage.ContentTypeHelper.ProcessDefaultTagAttributes(XmlTextReader reader)
   at System.IO.Packaging.ZipPackage.ContentTypeHelper.ParseContentTypesFile(ZipFileInfoCollection zipFiles)
   at System.IO.Packaging.ZipPackage.ContentTypeHelper..ctor(ZipArchive zipArchive, IgnoredItemHelper ignoredItemHelper)
   at System.IO.Packaging.ZipPackage..ctor(Stream s, FileMode mode, FileAccess access, Boolean streaming)
   at System.IO.Packaging.Package.Open(Stream stream, FileMode packageMode, FileAccess packageAccess, Boolean streaming)
   at System.IO.Packaging.Package.Open(Stream stream)
   at NuGet.ZipPackage.EnsureManifest()
   at NuGet.ZipPackage..ctor(String filePath, Boolean enableCaching)
   at NuGet.ZipPackage..ctor(String filePath)
   at Squirrel.UpdateManager.InstallHelperImpl.<CreateUninstallerRegistryEntry>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Squirrel.Utility.<LogIfThrows>d__35`1.MoveNext()
2016-03-21 12:48:59> Unhandled exception: System.AggregateException: One or more errors occurred. ---> System.Xml.XmlException: 'Default' tag requires a nonempty 'Extension' attribute. Line 10, position 4.
   at System.IO.Packaging.ZipPackage.ContentTypeHelper.ValidateXmlAttribute(String attributeName, String attributeValue, String tagName, XmlTextReader reader)
   at System.IO.Packaging.ZipPackage.ContentTypeHelper.ProcessDefaultTagAttributes(XmlTextReader reader)
   at System.IO.Packaging.ZipPackage.ContentTypeHelper.ParseContentTypesFile(ZipFileInfoCollection zipFiles)
   at System.IO.Packaging.ZipPackage.ContentTypeHelper..ctor(ZipArchive zipArchive, IgnoredItemHelper ignoredItemHelper)
   at System.IO.Packaging.ZipPackage..ctor(Stream s, FileMode mode, FileAccess access, Boolean streaming)
   at System.IO.Packaging.Package.Open(Stream stream, FileMode packageMode, FileAccess packageAccess, Boolean streaming)
   at System.IO.Packaging.Package.Open(Stream stream)
   at NuGet.ZipPackage.EnsureManifest()
   at NuGet.ZipPackage..ctor(String filePath, Boolean enableCaching)
   at NuGet.ZipPackage..ctor(String filePath)
   at Squirrel.UpdateManager.InstallHelperImpl.<CreateUninstallerRegistryEntry>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Squirrel.Utility.<LogIfThrows>d__35`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Squirrel.Update.Program.<Install>d__4.MoveNext()
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Squirrel.Update.Program.executeCommandLine(String[] args)
   at Squirrel.Update.Program.main(String[] args)
---> (Inner Exception #0) System.Xml.XmlException: 'Default' tag requires a nonempty 'Extension' attribute. Line 10, position 4.
   at System.IO.Packaging.ZipPackage.ContentTypeHelper.ValidateXmlAttribute(String attributeName, String attributeValue, String tagName, XmlTextReader reader)
   at System.IO.Packaging.ZipPackage.ContentTypeHelper.ProcessDefaultTagAttributes(XmlTextReader reader)
   at System.IO.Packaging.ZipPackage.ContentTypeHelper.ParseContentTypesFile(ZipFileInfoCollection zipFiles)
   at System.IO.Packaging.ZipPackage.ContentTypeHelper..ctor(ZipArchive zipArchive, IgnoredItemHelper ignoredItemHelper)
   at System.IO.Packaging.ZipPackage..ctor(Stream s, FileMode mode, FileAccess access, Boolean streaming)
   at System.IO.Packaging.Package.Open(Stream stream, FileMode packageMode, FileAccess packageAccess, Boolean streaming)
   at System.IO.Packaging.Package.Open(Stream stream)
   at NuGet.ZipPackage.EnsureManifest()
   at NuGet.ZipPackage..ctor(String filePath, Boolean enableCaching)
   at NuGet.ZipPackage..ctor(String filePath)
   at Squirrel.UpdateManager.InstallHelperImpl.<CreateUninstallerRegistryEntry>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Squirrel.Utility.<LogIfThrows>d__35`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Squirrel.Update.Program.<Install>d__4.MoveNext()<---
```

**Comment by unindented** [comment]
AppVeyor seems to be building fine (https://ci.appveyor.com/project/unindented/electron-installer-windows), so this may suggest the problem is in your environment.

The error says `Could not find a part of the path 'C:\Users\IEUser\AppData\Local\poopie\packages\RELEASES'`. Have you tried creating all the intermediate folders in `C:\Users\IEUser\AppData\Local`?

**Comment by eddievagabond** [comment]
The same issue occurs for me on a clean install of windows 10 on virtual box. No event logs are written. @johnjelinek , did you come up with anything to solve this issue

**Comment by unindented** [comment]
@eddievagabond have you checked that `C:\Users\<your user>\AppData\Local` exists?

**Comment by eddievagabond** [comment]
@unindented I have. The files exist as listed in the log. I was able to reproduce this with the example application as well.

**Comment by unindented** [comment]
@eddievagabond dang. No clue what's happening then.

**Comment by johnjelinek** [comment]
I gave up for the time-being.

**Comment by endlist** [comment]
Fwiw, I ran into this while using MacOS to try to run the build script.  When I checked the squirrel log on the failed install, I noticed the corrupt message shows up even for apps I know are installed properly (i.e. Atom and Slack), so that seems to be a red herring.  The actual issue is the XmlException, which is apparently a problem in Squirrel related to trying to create the installer on a non-Windows OS with files that don't have an extension:  https://github.com/electron/windows-installer/issues/12

Ran it on Windows instead and the installers work fine.
